photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataSet.scala

```diff
@@ -73,15 +78,29 @@ protected[ml] class RandomEffectDataSet(
     update(updatedActiveData, updatedPassiveDataOption)
   }
 
+  /**
+   *
+   * @return The Spark context
+   */
   override def sparkContext: SparkContext = activeData.sparkContext
```

Comment:
Same for activeData. That one is fairly important IMHO (active/passive), so even if this PR is not focused on improving comments, now would be a good time to document active/passive.